hack/ci/e2e.sh

```diff
@@ -45,49 +40,49 @@ install_kind() {
   export PATH="${TMP_DIR}/bin:${PATH}"
 }
 
-# build kubernetes / node image, e2e binaries
-build() {
+# build kubernetes / node image, e2e binaries, with bazel
+build_with_bazel() {
   # possibly enable bazel build caching before building kubernetes
   if [[ "${BAZEL_REMOTE_CACHE_ENABLED:-false}" == "true" ]]; then
     create_bazel_cache_rcs.sh || true
   fi
 
   # build the node image w/ kubernetes
   kind build node-image --type=bazel --kube-root="$(go env GOPATH)/src/k8s.io/kubernetes"
-
   # make sure we have e2e requirements
-  #make all WHAT="cmd/kubectl test/e2e/e2e.test vendor/github.com/onsi/ginkgo/ginkgo"
   bazel build //cmd/kubectl //test/e2e:e2e.test //vendor/github.com/onsi/ginkgo/ginkgo
 
   # ensure the e2e script will find our binaries ...
   # https://github.com/kubernetes/kubernetes/issues/68306
+  # TODO: remove this, it was fixed in 1.13+
```

Comment:
when 1.16 lands we should consider how we want to keep testing kind with old kubernetes versions, our long tail is getting longer. I'd love to support users as far back as possible, but at some point it's probably going to get ridiculous :thinking:

Reply:
Maybe we have to provide some extra support for some versions of EOL.

Reply:
we have some code in kind that deals with < 1.12 behavior, it's not so bad, but we can't continue to run presubmits for arbitrarily many kubernetes versions, at some point it's going to be prohibitive.

we might try something like a manual smoke test before releasing, or some limited periodic CI or ...